src/DynamoCoreWpf/UI/GuidedTour/GuidesValidationMethods.cs

```diff
@@ -299,12 +302,27 @@ private static void PackageManagerViewModel_PropertyChanged(object sender, Syste
                     //Put the name of the Package to be searched in the SearchTextBox
                     packageManagerViewModel.SearchText = AutodeskSamplePackage;
 
+                    searchPackagesLoaded = true;
+
+                    EnableNextButton(null, null, true, GuideFlow.FORWARD);
```

Comment:
As null is being passed for stepInfo and uiAutomationData, can we get rid of these parameters in the method?

Reply:
This method is also being called via reflection with these parameters in Step class.
![image](https://user-images.githubusercontent.com/89042471/145049274-b8d78db3-36a1-4417-95d2-d1a7815486f6.png)
If I remove it will get an exception.